### PR TITLE
Add label to execution_plan

### PR DESCRIPTION
**Pull request by iNecas**
The value of the label is determined by the `action#label`. The default
goes to the `class.name` but can be overriden, for example ActiveJob can
override it with the job class name.

### CODE REVIEW

File: lib/dynflow/active_job/queue_adapter.rb [hunk 1]
```diff
@@ -28,6 +28,10 @@ def plan(attributes)
         def run
           input[:job_class].constantize.perform_now(*input[:job_arguments])
         end
+
+        def label
+          input[:job_class]
```
Comment (dLobatog) [inline lib/dynflow/active_job/queue_adapter.rb]: I think it'd be nice to have `job_class` or `input[:job_class].constantize.label`. The reason is that if one has 2 jobs scheduled, for example SyncLDAPUsers(usergroup), I want to check on initialization if there's a job scheduled for that already. 

```
Dynflow.world.find_execution_plans :label => "SyncLDAPUser" 
```

is not good enough, as it assumes only 1 job will be scheduled of the same class at any point in time.

```
Dynflow.world.find_execution_plans :label => "sync-ldap-user-myfavoritegroup" 
```

is different and does not make that assumption (it allows me to iterate over all usergroups and only schedule syncs for the ones that are not scheduled). 

It'd be as simple as defining the label in the ActiveJob as 

```
def label
    "sync-ldap-users-#{@usergroup}"
end
```

Comment (iNecas) [inline lib/dynflow/active_job/queue_adapter.rb]: AFAIK you pass the args to the `perform` method of the active job: I don't see where in the active job class you would be able to define the label.

The label here is just do disctinguish differend kinds of jobs, but for your case, I believe there should be something on application layer to link the resources and jobs. We use foreman-tasks for it right now though locks. I believe we need a counterpart in foreman.

**Comment by iNecas** [comment]
@dLobatog I believe there is not more I can do here to support your case and that the current patch is good enough to provide you ability to achieve the recurring check. 

@adamruzicka would you mind having a look?

**Comment by adamruzicka** [comment]
Looks good and works as expected.

I noticed you added some `#rubocop:disable` and never enabled them again. Was it intentional?

**Comment by iNecas** [comment]
Yes, it was intentional.

**Comment by iNecas** [comment]
I'm merging this, as it will help us also performance wise. Let's leave other comments for further PRs

**Comment by iNecas** [comment]
dynflow-0.8.22 with this code pushed to rubygems

**Pull request merged**